rules/sinks/third_parties/sdk/oracle/java.yaml

```diff
@@ -7,791 +7,791 @@ sinks:
+      - "oracle-base.com/articles/8i/auditing"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]audit).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.ContainerEngine
     name: Oracle ContainerEngine
     domains:
-      - "containerengine.oracle.com"
+      - "oracle.com/in/cloud/cloud-native/container-engine-kubernetes/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]containerengine).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.Core
     name: Oracle Core
     domains:
-      - "core.oracle.com"
+      - "oracle.com/industries/financial-services/banking/core-banking/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]core).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.Database
     name: Oracle Database
     domains:
-      - "database.oracle.com"
+      - "oracle.com/in/database/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]database).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.DNS
     name: Oracle DNS
     domains:
-      - "dns.oracle.com"
+      - "oracle.com/in/cloud/networking/dns/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]dns).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.Email
     name: Oracle Email
     domains:
-      - "email.oracle.com"
+      - "oracle.com/in/cloud/networking/email-delivery/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]email).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.FileStorage
     name: Oracle FileStorage
     domains:
-      - "filestorage.oracle.com"
+      - "oracle.com/in/cloud/storage/file-storage/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]filestorage).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.Identity
     name: Oracle Identity
     domains:
-      - "identity.oracle.com"
+      - "oracle.com/in/security/identity-management/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]identity).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.LoadBalancer
     name: Oracle LoadBalancer
     domains:
-      - "loadbalancer.oracle.com"
+      - "oracle.com/in/cloud/networking/load-balancing/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]loadbalancer).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.ObjectStorage
     name: Oracle ObjectStorage
     domains:
-      - "objectstorage.oracle.com"
+      - "oracle.com/in/cloud/storage/object-storage/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]objectstorage).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.Auth
     name: Oracle Auth
     domains:
-      - "auth.oracle.com"
+      - "oracle.com/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]auth).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.CircuitBreaker
     name: Oracle CircuitBreaker
     domains:
-      - "circuitbreaker.oracle.com"
+      - "oracle.com/"
     patterns:
       - "(?i)(com[.]oracle[.]bmc[.]circuitbreaker).*"
     tags:
 
   - id: ThirdParties.SDK.Oracle.Common
     name: Oracle Common
     domains:
-      - "common.oracle.com"
+      - "oracle.com/en-us/iaas/tools/java/3.4.0/com/oracle/bmc/common"
```

Comment:
Can you confirm if this URL is correct? Also, the rule looks like an SDK common library instead of a product. Remove if needed.